integreat_cms/core/formats/de/formats.py

```diff
@@ -0,0 +1,7 @@
+"""
+This file contains locale formats to override the default ones.
+See :setting:`django:FORMAT_MODULE_PATH` and :attr:`~integreat_cms.core.settings.FORMAT_MODULE_PATH` for more information.
+"""
+
+#: The default formatting to use for displaying datetime fields in any part of the system when using the German locale.
+DATETIME_FORMAT = "d. F Y \\u\\m H:i \\U\\h\\r"
```

Comment:
At the moment, German and English are supported by the CMS, I think @JoeyStk started to translate the CMS into dutch as well, but this is quite outdated in the moment and not enabled for the users...

The reason I didn't add a separate file for English is because I was already satisfied with the default representation (e.g. `Oct. 9, 2023, 10:15 p.m.`), but if you think it makes sense, we could also replace this by e.g. `October 9, 2023 at 10:15 p.m.`...

Reply:
The CMS supports English and Dutch, too. Maybe we can differentiate the date representation based on the language set by the user? 🤔

Reply:
@ulliholtgrave Django does this already by default. I just wasn't happy with the default representation of the German dates :sweat_smile:

Reply:
So I guess it's fine then?